testsuites/syncgateway/functional/topology_specific_tests/multiple_sync_gateways/test_sg_replicate.py

```diff
@@ -157,10 +158,19 @@ def test_sg_replicate_basic_test(params_from_base_test_setup):
     if prometheus_enabled and sync_gateway_version >= "2.8.0":
```

Comment:
It would have been good if the condition is like this 
if sync_gateway_version >= "2.8.0":
     if prometheus_enabled:
          < your if condition logic >
    else
         < your else condition logic > 

That's fine for now,